emily_cron/app/services/deposit_processor.py

```diff
@@ -97,8 +97,9 @@ def process_rbf_transactions(
                     DepositUpdate(
                         bitcoin_txid=tx.bitcoin_txid,
                         bitcoin_tx_output_index=tx.bitcoin_tx_output_index,
-                        status=RequestStatus.FAILED.value,
+                        status=RequestStatus.RBF.value,
```

Comment:
nit: can we change also the log a :94, and the comment at :91? They still mention `FAILED`